### DISCUSSION TITLE
Strike API integration or similar?

### DISCUSSION THREAD

**Discussion by JMadisonTP** [btcpayserver/btcpayserver#4477] [Technical Support]
I have been pounding the pavement to onboard any to btc payments who will listen.  Biggest obstacle i get is, well how to i convert to fiat and the end of day. I know this is not the ideal situation as you want merchants to hold btc and create a circular economy, unfortunately this is always the case.  So Im wondering is there some way for btcpay to work with say Strike api or any other exchange, that would help the merchant with the last mile use? Just a thought.

**Comment by bolatovumar** [comment]
> I have been pounding the pavement to onboard any to btc payments who will listen. Biggest obstacle i get is, well how to i convert to fiat and the end of day. I know this is not the ideal situation as you want merchants to hold btc and create a circular economy, unfortunately this is always the case. So Im wondering is there some way for btcpay to work with say Strike api or any other exchange, that would help the merchant with the last mile use? Just a thought.

This is possible through BtcTransmuter already: https://docs.btcpayserver.org/Transmuter/

![Capture](https://user-images.githubusercontent.com/1934678/209422578-47ee89cf-473a-4fbf-8c99-0dc5965c1336.PNG)

**Reply by JMadisonTP** [reply]
Oh wow thanks, I'll check it out

**Reply by JMadisonTP** [reply]
there is no strike integration on there. Im guessing Kraken probably next best choice since they do accept LN in and out (right?)

**Reply by bolatovumar** [reply]
> Im guessing Kraken probably next best choice since they do accept LN in and out (right?)

@JMadisonTP yeah, Kraken is your best bet. I know others have set up Transmuter to work with Kraken.

**Comment by pavlenex** [comment]
https://github.com/Marfusios/strike-btcpayserver-plugin
